ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -61,7 +85,7 @@ AdvancedGasControls.propTypes = {
   onManualChange: PropTypes.func,
   gasLimit: PropTypes.number,
   setGasLimit: PropTypes.func,
-  gasPrice: PropTypes.string,
```

Comment:
this should be `number` and not `string` right?

Reply:
Hmm not sure the component that is using it is also propTyped as a string so I think we can just leave it.

Reply:
i changed it to a number, will revert the changes now, thanks for your input

Reply:
Reverted changes here [`1ba4483`](https://github.com/MetaMask/metamask-extension/pull/19758/commits/1ba44835e6a04b9731f1b5304816f148bccf20b4)